vita3k/gui/src/pkg_install_dialog.cpp

```diff
@@ -74,6 +74,7 @@ void draw_pkg_install_dialog(GuiState &gui, EmuEnvState &emuenv) {
     ImGui::SetNextWindowSize(ImVec2(616.f * SCALE.x, 264.f * SCALE.y));
     if (ImGui::BeginPopupModal("install", nullptr, ImGuiWindowFlags_AlwaysAutoResize | ImGuiWindowFlags_NoDecoration)) {
         ImGui::SetWindowFontScale(RES_SCALE.x);
+        ImGui::PushStyleVar(ImGuiStyleVar_ChildRounding, 10.f * SCALE.x);
```

Comment:
Problème here is wrong, resuest postale en this is set Before beginchild only

Reply:
i revert it.